backend/stakepoold/server.go

```diff
@@ -9,6 +9,7 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"net/http" // promhttp
```

Comment:
> // promhttp

It isn't clear what this comment is trying to convey. Could you please either remove or add more detail?

Reply:
sorry about that, this has been removed